libraries/bxdf/disney_brdf_2012.mtlx

```diff
@@ -1,6 +1,6 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <materialx version="1.36" cms="ocio" colorspace="lin_rec709">
-  <nodedef name="ND_disney_brdf_2012_surface" type="surfaceshader" node="disney_brdf_2012">
```

Comment:
Tons of changes like this one which remove the type in favor of a child output called "out".